content/blog/peace-of-mind-with-cloud-secret-providers/index.md

```diff
@@ -0,0 +1,591 @@
+---
+title: "Peace of Mind with Cloud Secret Providers"
+
+date: 2020-04-29T14:26:35-07:00
+draft: true
+meta_desc: "Learn how to encrypt your Pulumi secrets with a cloud secrets provider to ensure secrets never leave your control"
```

Comment:
meta_desc is too long, I think it's limited to 60 char by Hugo

Reply:
Fixed!